### PR TITLE
Updated FLYWOOF405NANO unofficial target

### PR DESCRIPTION
**Pull request by wchnflr**
Add UART2
Add ICM42605 gyro which is compatible with ICM-42688-P gyro on this FC

### CODE REVIEW

File: src/main/target/FLYWOOF405NANO/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+target_stm32f405xg(FLYWOOF405NANO)
```
Comment (mmosca) [inline src/main/target/FLYWOOF405NANO/CMakeLists.txt]: Can you add SKIP_RELEASES?

Comment (wchnflr) [reply src/main/target/FLYWOOF405NANO/CMakeLists.txt]: Done.

**Comment by mmosca** [comment]
Have you tried the FLYWOOF405PRO target?

**Comment by wchnflr** [comment]
> Have you tried the FLYWOOF405PRO target?

Yes, I have since been successful in flashing the FLYWOOF405PRO target onto this FC and configured it to work.

**Pull request closed without merging**